lib/testing/src/lib/protractor/process-services-cloud/pages/edit-process-filter-cloud-component.page.ts

```diff
@@ -119,6 +133,23 @@ export class EditProcessFilterCloudComponentPage {
         return this.processDefinitionNameDropdown.selectDropdownOption(option);
     }
 
+    setSuspendedDateRangeDropDown(option: string): Promise<void> {
+        return this.suspendedDateRangeDropdown.selectDropdownOption(option);
```

Comment:
SelectDropdownOption does not return anything, why return with Promise<void>? 
https://github.com/Alfresco/alfresco-ng2-components/blob/13171710d004cefad091eafa9dd17c0bf53a6cd5/lib/testing/src/lib/protractor/core/pages/material/dropdown.page.ts#L87

Reply:
Technically it does, since any async function in JavaScript always returns a Promise, whether it's explicit or implicit. Here the lack of return statement in `selectDropdownOption` implicitly means it's returning `Promise<void>`.